lib/codegen/Cargo.toml

```diff
@@ -16,6 +16,7 @@ failure = { version = "0.1.1", default-features = false, features = ["derive"] }
 failure_derive = { version = "0.1.1", default-features = false }
 hashmap_core = { version = "0.1.9", optional = true }
 target-lexicon = { version = "0.0.3", default-features = false }
+log = { version = "0.4.3", default-features = false}
```

Comment:
The `dbg!` macro had a way to disable itself in release builds, which was controlled by `cfg!(debug_assertions)`. Fortunately, it looks like the `log` crate can do this to, and is more flexible: https://docs.rs/log/0.4.3/log/#compile-time-filters . What would you think about setting either `release_max_level_info` or maybe `release_max_level_warn` here, so that most log messages are disabled in release builds?

Whatever we do here, we should do the same thing for the wasm and module crates too.